components/inputs/docs/input-checkbox.md

```diff
@@ -135,3 +127,11 @@ As an alternative to using the `<d2l-input-checkbox>` custom element, you can st
 </script>
 <d2l-my-checkbox-elem></d2l-my-checkbox-elem>
 ```
+
+## Accessibility
+
+The `d2l-input-checkbox` component follows W3C's best practice recommendations for a [checkbox](https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/)
```

Comment:
```suggestion
The `d2l-input-checkbox` component follows W3C's best practice recommendations for a [checkbox](https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/).
```

Reply:
Is this adding a period to a bullet point? No need for periods unless there are multiple sentences. This is best practice, though we're not consistent with this in our docs today 😬